src/calculateRentalCost.js

```diff
@@ -3,8 +3,25 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const dayRentCost = 40;
+
+  const longTerm = 7;
+  const shortTerm = 3;
+
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+
+  if (days >= longTerm) {
+    return days * dayRentCost - longTermDiscount;
```

Comment:
The 'if' statement correctly uses a return, which simplifies the logic and avoids the need for an 'else' statement. This follows the best practices outlined in the checklist.